README.md

```diff
@@ -25,22 +25,23 @@ Consider adding this to your configuration.
   :ensure t
   :hook (text-mode . (lambda ()
                        (require 'eglot-ltex)
-                       (call-interactively #'eglot)))
+                       (eglot-ensure)))
   :init
   (setq eglot-languagetool-server-path "path/to/ltex-ls-XX.X.X/"))
 ```
 
 ## :wrench: Configuration
 
-Create `.dir-locals.el` file in the the project root directory.
+Create `.dir-locals.el` file in the project root directory to configure the ltex language server, for example:
 
 ```el
-((nil
-  (eglot-workspace-configuration
-   . ((ltex-ls . ((language . "en-US")))))))
+((nil .
+  ((eglot-workspace-configuration
+    . (:ltex-ls  (:language  "en-US"
+                  :additionalRules (:motherTongue "de-DE")))))))
 ```
 
-*P.S. See all possible configuration [here](https://valentjn.github.io/vscode-ltex/docs/settings.html).*
+You can find all possible ltex configuration options [here](https://valentjn.github.io/vscode-ltex/docs/settings.html). See the [Eglot manual](https://joaotavora.github.io/eglot/#JSONRPC-objects-in-Elisp) on how to translate the LSP options into the property-list format that Eglot requires.
```

Comment:
I prefer no more than 80 columns, can you fix this for me? Thanks!

Reply:
Resolved by 0b8efceda27804fa696b610dd9a7e62bd2e83967